### ISSUE TITLE
Add an aria-expanded attribute to the flyout button

### ISSUE THREAD

**Issue by Veyfeyken** [a-ui/core_branding_scss#156]
### Problem

[atoms/atoms.flyout](https://a-ui.github.io/core_branding_scss/#molecules-flyout)

The button that triggers the flyout is a regular button. A screenreader user cannot know the status (open or closed) of the component.

![screenshot flyout](https://user-images.githubusercontent.com/870049/98664292-799fcd00-234a-11eb-8eb1-431ea4beda1a.jpg)

### Solution

* Add an aria-expanded attribute as described in the ARIA design pattern for disclosure.

### Articles and links

* [ARIA Design pattern and widgets: Disclosure (Show/Hide)](https://www.w3.org/TR/wai-aria-practices-1.2/#disclosure)
* [Accessible Rich Internet Applications (WAI-ARIA) 1.2: aria-expanded (state)](https://www.w3.org/TR/wai-aria-1.2/#aria-expanded)

**Comment by TriangleJuice** [comment]
Hey @Veyfeyken this issue should be fixed in the next iteration of our core branding (`v4.3.0`)

**Comment by Veyfeyken** [comment]
The aria-expanded attribute is present but needs to be on the button to work properly. It always has to be on the element that initiates the action, not on the content that expands. 

Current:

```html
<div class="m-flyout is-open">
  <button class="a-button">Default flyout</button>
  <div class="m-flyout__content has-padding" aria-expanded="true">
    <p>The default flyout will never be bigger than the element that triggers it.</p>
  </div>
</div>
```

Suggested:
```html
<div class="m-flyout is-open">
  <button class="a-button" aria-expanded="true">Default flyout</button>
  <div class="m-flyout__content has-padding">
    <p>The default flyout will never be bigger than the element that triggers it.</p>
  </div>
</div>
```
